### DISCUSSION TITLE
Custom sound for notifications in IOS

### DISCUSSION THREAD

**Discussion by josuelmm** [capawesome-team/capacitor-firebase#526] [General]
Good day, how do I configure a custom sound for notifications on iOS. On Android it works fine, because from the backend I send the channel and sound, that is, setSound('sos'), setChannelId('sos')
and it plays the sound, but on Apple it always sounds something generic and I have not been able to customize it. On Android the file is called sos.mp3 and it plays.

Channel example, but this work only in Android
`let dataChannel:any = {
      description: "",
      id: "default",
      name: "Default",
      importance: 4,
      lightColor: "#000000",
      lights: true,
      visibility:1,
      vibration: true,
      sound:sonido
    };
    await FirebaseMessaging.createChannel(dataChannel).then((data:any) =>{}).catch((err) => {});`

**Comment by robingenz** [comment]
Nothing special here for Capacitor. This post should help you: https://stackoverflow.com/a/58974220/6731412
